deck/models.py

```diff
@@ -1,3 +1,4 @@
+from datetime import datetime
```

Comment:
The `now()` should be imported from the timezone instead of the datetime module:
```
>>> from django.utils import timezone
>>> timezone.now()
````